test-crates/pyo3-pure/Cargo.toml

```diff
@@ -2,8 +2,6 @@
 authors = ["konstin <konstin@mailbox.org>"]
 name = "pyo3-pure"
 version = "2.1.2"
-description = "Implements a dummy function (get_fortytwo.DummyClass.get_42()) in rust"
```

Comment:
I'd keep both cargo and PEP 621 metadata here. It's redundant, but `cargo publish` can't read python metadata and vice versa.

Reply:
The `package.metadata` parts below should be removed though